README.md

```diff
@@ -566,6 +566,62 @@ words from the list by modifying the file.
 Rejected words automatically get flagged by Vale. To modify the list of rejected words, modify the
 [reject.txt](/vale/styles/Vocab/Internal/reject.txt) file.
 
+## Check Formatting
+
+We use [Prettier](https://prettier.io/) to maintain uniform and consistent formatting across the docbase. When you
+commit changes, Prettier formats the staged files automatically. Then, once you create a pull request, it verifies that
+all files comply with our Prettier configuration.
+
+**Note:** The build fails if the Code Formatting check doesn't pass.
+
+To manually check the formatting before pushing your work upstream, execute the following command in your terminal:
+
+```
+npm run format-check
+```
+
+If you receive an output similar to the following, the Code Formatting check has failed.
+
+```
+Checking formatting...
+[warn] README.md
+[warn] Code style issues found in the above file. Run Prettier to fix.
+```
+
+In this case, issue the following command to format all files manually.
+
+```
+npm run format
```

Comment:
Can we use the Makefile instead? That's a pattern we started a long time ago.

Reply:
I've added the Make command. Could you please document this pattern so that it's explicit for anyone who wants to introduce changes to the docbase?